snap/snapcraft.yaml

```diff
@@ -8,11 +7,17 @@ confinement: classic
 
 apps:
     heroku:
-        command: bin/heroku
+        command: bin/node $SNAP/lib/node_modules/heroku-cli/bin/run.js
 
 parts:
-  hello-node-snap:
+  heroku:
+    source: .
     plugin: nodejs
-    node-engine: 7.10.0
-    node-packages:
-      - heroku-cli
```

Comment:
This this code is in the source I thought it would be better;
Given this change was accepted, and iif you use annotated git tags then changing `version: '6.13.1'` to `version: git` will set the version nicely.